public/app/features/dashboard-scene/saving/provisioned/DashboardPreviewBanner.tsx

```diff
@@ -1,37 +1,43 @@
 import { Alert, Icon, Stack } from '@grafana/ui';
 import { DashboardPageRouteSearchParams } from 'app/features/dashboard/containers/types';
+import { DashboardRoutes } from 'app/types';
 
 interface DashboardPreviewBannerProps {
   queryParams: DashboardPageRouteSearchParams;
+  route?: string;
+  path?: string;
 }
-export function DashboardPreviewBanner({ queryParams }: DashboardPreviewBannerProps) {
-  if (!queryParams.isPreview) {
+
+export function DashboardPreviewBanner({ queryParams, route, path }: DashboardPreviewBannerProps) {
+  const hasPrLink = Boolean(queryParams.prLink);
+  const isProvisioned = route === DashboardRoutes.Provisioning && path;
+
+  if ('kiosk' in queryParams || (!queryParams.isPreview && !isProvisioned)) {
```

Comment:
This means that we will only show this banner for provisioned routes, which is not the case when creating a new dashboard?